.ci/pitest.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+# Attention, there is no "-x" to avoid problems on CircleCI
+set -e
+
+function list_profiles() {
+  POM_PATH="$(dirname "${0}")/../pom.xml"
+  cat $POM_PATH | sed -n -e 's/^.*<id>\(pitest-[^<]\+\)<\/id>.*$/\1/p' | sort
+}
+
+case $1 in
+
+--list)
+  echo "Supported profiles:"
+  list_profiles "${0}"
+  ;;
+
+*)
+  PROFILES=$(list_profiles "${0}");
+
+  if [[ $(echo $PROFILES | grep -w -- ${1} | cat) != "" ]]; then
```

Comment:
read 
 `--` - https://stackoverflow.com/a/2427987
`-w` - https://stackoverflow.com/a/8063284

Reply:
Isn't 
```bash
if [[ $(echo "$PROFILES" | grep -w "${1}") != "" ]]; then
```
sufficient?

`--` is not required as `$1` would be `pitest-XYZ` not something like `--pitest-XYZ`, also `cat` can also be omitted, it is already printed to output stream.

Reply:
Without `--` , `--lis` is failing in weird way.

Cat is good to avoid failure is grep found nothing.

Reply:
Good points, resolved.